src/export_data.py

```diff
@@ -0,0 +1,122 @@
+"""Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import argparse
+import ee
+from utils import TaskManager, SampleExporter, stack_bands_from_imagery
+from utils import RoadImagery, MultiSpectralImagery, TropomiImagery
+from utils import DSMImagery, WindImagery
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Export Data from EarthEngine'
+                                     )
+    parser.add_argument('--params_path', type=str,
+                        help="json parameters file location")
+    parser.add_argument('--export_folder', type=str,
+                        help="Cloud export folder")
+    parser.add_argument('--bucket', type=str,
+                        help="bucket name",
+                        default='aghriss-air-quality')
+    parser.add_argument('--num_samples', type=int,
+                        help="number of samples to export per image",
+                        default=1000)
+    parser.add_argument('--num_shards', type=int,
+                        help="number of sharding of samples before export",
+                        default=10)
+
+    args = parser.parse_args()
+
+    params = json.load(open(args.params_path, 'r'))
+    BUCKET = args.bucket
+    DIRECTORY = args.export_folder
+    num_samples = args.num_samples
+    num_shards = args.num_shards
+
+    print("Loading", (args.params_path))
+    params_path = args.params_path
+    params = json.load(open(params_path, 'r'))
+    PATCH_BANDS = []
+    PATCH_BANDS += params['bands']['multispectral']
+    PATCH_BANDS += params['bands']['tropomi']
+    PATCH_BANDS += params['bands']['road'] + params['bands']['dsm']
+    STACKED_WIND_BANDS = ["%i_%s" % (i, b) for b in params['bands']['wind']
+                          for i in range(12)]
+    PATCH_BANDS += STACKED_WIND_BANDS
+    BANDS = PATCH_BANDS + ["HOD", "DOW", "DOM", "MOY", "latitude", "longitude",
+                           "valid"]
+    SCALE = params['scale']
+    KERNEL_RADIUS = params['kernel_radius']
+    # Initialize EE
+    ee.Authenticate()
+    ee.Initialize()
+    # Creating diffrent kernels
```

Comment:
While we are at it, can you add a bit more int he comment about why we are making different kernels? And what they are and do?